glib/src/collections/strv.rs

```diff
@@ -1151,7 +1241,8 @@ impl<const N: usize> IntoStrV for [GString; N] {
                 let mut strs = ptrs.add(self.len() + 1) as *mut c_char;
 
                 for (i, item) in self.iter().enumerate() {
-                    ptr::copy_nonoverlapping(item.as_ptr(), strs, item.len() + 1);
+                    ptr::copy_nonoverlapping(item.as_ptr() as *const _, strs, item.len());
+                    *strs.add(item.len()) = 0;
                     *ptrs.add(i) = strs;
                     strs = strs.add(item.len() + 1);
```

Comment:
Should we care about alignment?

Reply:
char arrays have an alignment of 1 so that's fine here. But that's exactly why the pointers come first and afterwards the characters :)